### PR TITLE
Daos 6338 SWLC: Change DAOS license text #7

### PR DESCRIPTION
**Pull request by GitHuaKuang**
Found few missed files to update.

    Skip-build: true
    Skip-test: true

### CODE REVIEW

File: utils/rpms/daos.spec [hunk 1]
```diff
@@ -9,7 +9,7 @@ Version:       1.1.2.1
 Release:       6%{?relval}%{?dist}
 Summary:       DAOS Storage Engine
 
-License:       Apache
+License:       BSD-2-Clause-Patent
```
Comment (jolivier23) [inline utils/rpms/daos.spec:12]: You need to bump the release on line 9 and add a change log entry

**Review by jolivier23 — CHANGES_REQUESTED** [review]

File: utils/rpms/daos.spec (hunk 1 above)
Comment (GitHuaKuang) [reply utils/rpms/daos.spec:12]: Oops, yes.

**Review by brianjmurrell — COMMENTED** [review]
Does debian/copyright need updating also?

**Comment by GitHuaKuang** [comment]
> Does debian/copyright need updating also?

If the file currently have license statement of using Apache, we need to change to the new one. I searched all files in daos that contains "Apache", didn't see debian files, do you know any particular files need to be changed?
The related Copyright date update is more due to the commit to the file, our pre-commit will try to update that to those to be committed files.

**Comment by brianjmurrell** [comment]
> If the file currently have license statement of using Apache, we need to change to the new one. I searched all files in daos that contains "Apache", didn't see debian files, do you know any particular files need to be changed?

You need to look at the actual file.  It is a license statement for the project.  It doesn't currently say Apache.  I would imagine it needs updating.

**Comment by GitHuaKuang** [comment]
> > If the file currently have license statement of using Apache, we need to change to the new one. I searched all files in daos that contains "Apache", didn't see debian files, do you know any particular files need to be changed?
> 
> You need to look at the actual file. It is a license statement for the project. It doesn't currently say Apache. I would imagine it needs updating.

OK, just realized ./debian/copyright is the actual file you pointed to (I though you mean all files for debian and their Copyright statement).

This is the license in this file:
License: Intel_Open_Source_License

And the description of IOSL from https://en.wikipedia.org/wiki/Intel_Open_Source_License :
The Intel Open Source license is identical to the BSD license with the following section added:
EXPORT LAWS:
...
Intel has voluntarily retracted the license from the OSI list of open source licenses to prevent license proliferation. Intel has ceased to use or recommend this license.

So I think it's safe to change it to BSD-2-Clause-Patent.

File: debian/copyright [hunk 2]
```diff
@@ -3,9 +3,9 @@ Upstream-Name: cart
 Source: https://github.com/daos-stack/cart
 
 Files: *
-Copyright: Copyright 2016-2019 Intel Corporation <daos@daos.groups.io>
-License: Intel_Open_Source_License
- Copyright (C) 2016-2019 Intel Corporation
+Copyright: Copyright 2016-2021 Intel Corporation <daos@daos.groups.io>
```
Comment (jolivier23) [inline debian/copyright:6]: @brianjmurrell why does line 3 reference old cart repo?

**Review by jolivier23 — APPROVED** [review]

**Review by cibervicho — DISMISSED** [review]

**Review by jolivier23 — CHANGES_REQUESTED** [review]
You lost a race on daos.spec so you'll need to update to new version

**Review by jolivier23 — APPROVED** [review]

File: debian/copyright (hunk 2 above)
Comment (brianjmurrell) [reply debian/copyright:6]: Copy-n-pasto I'd guess.

**Review by mjmac — DISMISSED** [review]
LGTM

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
Note that in debian/copyright, there are multiple licences with path globs so that one can have different licenses for different files.  Currently, while that file is saying BSD-2-Clause-Patent for `*`, it is also then overriding that and saying GPL-2+ for for `debian/*`: https://github.com/daos-stack/daos/blob/9c65a48a464559a4c944d1fc9b06ecce4b34feb0/debian/copyright#L47-L64

Is this really what we want?  If not, delete lines 45 to the end of the file.

Note that if you do modify `debian/copyright` to fix the above, it would be worthwhile changing `cart` on line 2: https://github.com/daos-stack/daos/blob/9c65a48a464559a4c944d1fc9b06ecce4b34feb0/debian/copyright#L2 to `daos`.

You should also add a new entry to `debian/changelog`, bumping up the _release_ there the same way you did with the `daos.spec` file.

**Review by cibervicho — DISMISSED** [review]

**Review by jolivier23 — APPROVED** [review]

**Review by cibervicho — APPROVED** [review]

**Pull request merged**